src/Control/Inciter/InputDeck/Grammar.h

```diff
@@ -163,10 +163,44 @@ namespace grm {
              p0.size() != problem.size() )
           Message< Stack, ERROR, MsgKey::VORTICAL_UNFINISHED >( stack, in );
       }
+      else if (problem.back() == inciter::ctr::ProblemType::NL_ENERGY_GROWTH) {
```

Comment:
The `nonlinear energy growth` and the `Rayleigh-Taylor` test cases are newly being added and they are not final (and not tested) at this point. Will work these next.